web/profiles/joinup/src/Form/SolutionsByLicenceForm.php

```diff
@@ -99,8 +99,15 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     ];
 
     $licence_id = $form_state->getValue('licence_option');
-    $results = $this->connection->query($this->getCountQuery($licence_id));
-    $total = reset($results)->total->getValue();
+
+    // Under some unknown circumstances the result returned can be a string with
+    // the value 'binding' instead of the expected count. This has been causing
+    // random test failures, and has also been observed during debugging on a
+    // development environment. It appears that calling ::getArrayCopy() works
+    // around this somehow and the problem can no longer be replicated.
+    $results = $this->connection->query($this->getCountQuery($licence_id))->getArrayCopy();
+    $result = reset($results);
+    $total = $result->total->getValue();
```

Comment:
This is an unrelated bugfix. This caused a random failure during development of this PR.